content/blog/2021/07/camel-k-release-1-5/index.md

```diff
@@ -0,0 +1,61 @@
+---
+title: "Camel K 1.5 release"
+date: 2021-07-13
+draft: false
+authors: [squakez]
+categories: ["Releases", "Camel K"]
+preview: "What's new in Camel K 1.5"
+---
+
+[Camel K 1.5 version](https://github.com/apache/camel-k/releases/tag/v1.5.0) is public since a few days. We're proud to announce this new release containing enhancements and fixes. We've worked hard to fix the most important issues discovered with an eye on stabilization and performances. This new release is based on **Camel Quarkus 2.0** and **Apache Camel 3.11**.
+
+Let's find out what's new in Camel K 1.5!
+
+## New Camel Quarkus and Quarkus versions
+
+**Camel Quarkus** runtime has become the building block on which the `Integration` is materialized. With the new 2.0 version out, there are great improvements which are leveraging the major release of **Quarkus 2.0**. We're inheriting all the good stuff in performance and components coverage that is introduced lately in this new version.
+
+* [Camel Quarkus 2.0.0 released](/blog/2021/06/camel-quarkus-release-2.0.0/)
+* [What's new in Camel 3.11.0](/blog/2021/06/Camel311-Whatsnew/)
+
+## Tune your logs
+
+As software developers, we are daily watching logs to discover what's going on for a piece of software that is failing. **Observability** and all the activities around that area are needed in order to have a reliable software solution. We know that, and we've introduced a new `trait` that will simplify the work of tuning and integrating the log streams in your logging systems.
+
+You can learn more in [a blog dedicated to the topic](/blog/2021/05/new-camel-k-logging-features/).
+
+## S3 dependencies
+
+**S3** protocol is often used as a way to store Maven dependencies. With version 1.5 you will be able to configure your Camel K to retrieve such dependencies.
+
+## Configuration refactoring
+
+Prior to this version, we may have found a bit clumsy the work of providing a configuration file to our `Integration`. As a developer, you will now be able to define in details how and where to load a configuration file, filter a `Configmap` or a `Secret` and also distinguish between __build-time__ and __runtime__ properties. An entire blog covering this feature will be soon available, stay tuned!
+ 
+## More Kamelet commands
+
+Since their appearence, **Kamelets** have gained quite a lot of traction. With this new release we've introduced new commands in the `kamel` CLI to help you discovering and managing them. You will have a new command available `kamel kamelet` that will help you listing and deleting any `Kamelet`. Also `kamel describe kamelet` has been introduced to help you analyising the available Kamelets.
```

Comment:
```suggestion
Since their appearence, **Kamelets** have gained quite a lot of traction. With this new release we've introduced new commands in the `kamel` CLI to help you discovering and managing them. You will have a new command available `kamel kamelet` that will help you listing and deleting any `Kamelet`. Also `kamel describe kamelet` has been introduced to help you analysing the available Kamelets.
```